src/client/app/core/location-filter/location-filter.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.core')
+        .controller('LocationFilterController', LocationFilterController);
```

Comment:
now, this file should be renamed to `location-filter.controller.js` because it's confusing whether it's a filter or a controller